src/forge/rsync.py

```diff
@@ -104,6 +151,11 @@ def _rsync(config, ip):
             for ip, _ in targets:
                 logger.info('Rsync destination is %s', ip)
                 rval = _rsync(config, ip)
+
+                if config.get('s3_path'):
```

Comment:
Should we allow users to have just s3_path and not rsync_path?

Reply:
We can, I just wasn't sure if that's what we wanted.

Reply:
What do you think @npatel-cars ?
Is there a use-case for remote-only rsync?

Reply:
I talked to Nikhil and I've modified the PR to allow for S3 only rsyncs. This does introduce a slight regression to the required arguments checker, but for how we check the required args I don't think there's a really good way to do an 'or' argument. That's something that we can look to add, maybe with the type annotations like `Union`. It would also be interesting to also look how that can be better implemented with the upcoming configuration change.